helpers/app_helper.rb

```diff
@@ -109,7 +110,7 @@ def find_id
     until answered
       puts 'Please Select a person using the selection number for the person (number to the left of the name)'
       @rentals.map(&:person).to_set.each_with_index do |person, idx|
-        puts "Selection #{idx} - #{person.name}"
+        puts "Selection #{idx} - #{person[:name]}"
```

Comment:
-  [ ] Nice working on your helper methods, kindly check this `block` i get the error in the screenshot when I try to List all rentals for a given person id.

![image](https://user-images.githubusercontent.com/48371156/178097220-d3fe82ad-09a0-4718-84bd-1ac46b1a5879.png)

Reply:
fixed